app/javascript/components/pages/SpotIndex.js

```diff
@@ -1,9 +1,53 @@
 import React, { Component } from 'react'
+import { Card, CardImg, CardBody, CardTitle, CardSubtitle, Button, Col, Row } from 'reactstrap'
 
 class SpotIndex extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    conditionalRendering = () =>{
```

Comment:
This is a cool approach.  I was more implying having only one section be for logged in and one section for logged out.